src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -474,7 +474,7 @@ void UnitConverterViewModel::OnButtonPressed(Platform::Object ^ parameter)
 
     static constexpr UCM::Command OPERANDS[] = { UCM::Command::Zero, UCM::Command::One, UCM::Command::Two,   UCM::Command::Three, UCM::Command::Four,
                                                    UCM::Command::Five, UCM::Command::Six, UCM::Command::Seven, UCM::Command::Eight, UCM::Command::Nine };
-	if (m_isInputBlocked)
+    if (m_isInputBlocked && (command != UCM::Command::Clear && command != UCM::Command::Backspace))
```

Comment:
Minor nit:
You've got an extra set of parens here.  I understand that you are adding them for clarity, but I think in this case, it's straightforward enough that we don't need them.  Also, for readability, I like having each condition on a separate line if there are more than 2 conditions.

Reply:
This is done.